.test_pull_requests.json.sample

```diff
@@ -21,22 +21,50 @@
       "name": "test",
       "test_prefix": "Test Results:",
       "allow_multiple": true,
-      "branches": { "*": {"jenkins_job_name": "test_pull_requests_stage",
-                              "downstream_job_name": "build_image_stage",
-                              "build_token": "mytoken"},
-                    "master": {"jenkins_job_name": "test_pull_requests",
-                               "downstream_job_name": "build_image",
-                               "build_token": "mytoken"
-                               "addtl_jenkins_params": {"group1": {"param1": "value1",
-                                                                   "param2": "value2"} }
-                              } },
+      "branches": {
+        "*": {
+          "jenkins_job_name": "test_pull_requests_stage",
+          "downstream_job_name": "build_image_stage",
+          "build_token": "mytoken"
+        },
+        "master": {
+          "jenkins_job_name": "test_pull_requests",
+          "downstream_job_name": "build_image",
+          "build_token": "mytoken",
+          "addtl_jenkins_params": {
+            "group1": {
+              "param1": "value1",
+              "param2": "value2"
+            }
+          }
+        }
+      },
       "repo_to_teams": {
-        "myrepo1": ["111111", "222222", "333333"],
-        "myrepo2": ["111111", "444444"]
+        "myrepo1": [
+          "111111",
+          "222222",
+          "333333"
+        ],
+        "myrepo2": [
+          "111111",
+          "444444"
+        ]
+      },
+      "repo_to_admin_teams": {
+        "myrepo1": [
+          "111111"
+        ],
+        "myrepo2": [
+          "111111"
+        ]
       },
       "repo_to_pull_id_param": {
         "myrepo1": "MYREPO1_PULL_ID",
         "myrepo2": "MYREPO2_PULL_ID"
+      },
+      "flake_config": {
```

Comment:
The word flake shows up too many times in this map.
And don't love _config inside a config file.  It's redundant.